### ISSUE TITLE
Use a hyper-legible font in NVDA

### ISSUE THREAD

**Issue by seanbudd** [nvaccess/nvda#17356]
### Is your feature request related to a problem? Please describe.

To greater improve the accessibility of the NVDA GUI, a hyper-legible font  should be used.

### Describe the solution you'd like
Example font: https://fonts.google.com/specimen/Atkinson+Hyperlegible

This font is developed by the [Braille Institute](https://www.brailleinstitute.org/freefont/)
>Our font, Atkinson Hyperlegible was carefully developed by the Braille Institute to help low-vision readers. It improves legibility and readability through clear, and distinctive letters and numbers.

It helps improve readibility for all users, such as more easily distinguishing 1, I, l, (see also #16438)

### Describe alternatives you've considered
Different font options?

**Comment by ManolisPapa** [comment]
I believe this can be done in a brute force manner by adding a self.SetFont() to every GUI related class. As a visually impaired person I see a slight difference when using Atkinson-Hyperlegible and reading without my glasses so it could be worth a try. I would like to work on this but I can't think of any workaround to manually setting the font for each class. Any thoughts?

**Comment by seanbudd** [comment]
@ManolisPapa - I ran into a similar conclusion. After reading wxWidgets forums seems like that is the only option. We'd probably have to create our own subclasses of every wx control and add a MixIn which sets the font.

**Comment by ManolisPapa** [comment]
@seanbudd - I think I found a way to make this work. I added a font option so that users can select any font that's downloaded in their systems and added it to the config profile. After that I went through most gui related classes and added a self.SetFont. Luckily all of these classes that I worked on (I haven't implemented all of them yet) inherit from wx.Panel or wx.Dialog and thus have a SetFont method. I will open a WIP PR and let you have a look and see if this is a good way forward.
